hw02-generics/src/main/java/homework/CustomerService.java

```diff
@@ -0,0 +1,43 @@
+package homework;
+
+import java.util.Comparator;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.Objects;
+import java.util.TreeMap;
+
+public class CustomerService {
+
+    private final TreeMap<Customer, String> customerSortedByScoresMap =
```

Comment:
Лучше тип объявлять как интерфейс:
```suggestion
    private final NavigableMap<Customer, String> customerSortedByScoresMap =
```